### DISCUSSION TITLE
为什么这个项目下载的MPV.EXE 主程序我打不开。调用别人的MPV.EXE复制过来就能打开。

### DISCUSSION THREAD

**Discussion by rong2er** [hooke007/mpv_PlayKit#316] [回收]
为什么这个项目下载的MPV.EXE 主程序我打不开。调用别人的MPV.EXE复制过来就能打开。

**Comment by hooke007** [comment]
看说明

**Reply by rong2er** [reply]
7. 新版FAQ
Q: 如何在旧版硬件平台上使用mpv-lazy（或win7系统上使用vs滤镜）？
A: 自行下载 [shinchiro版常规mpv主程序](https://github.com/shinchiro/mpv-winbuild-cmake/releases) 替换（使用Python3.8embedded替换）。

是这个吗？是不是必须要下载替换主程序

**Reply by hooke007** [reply]
第一行就写了。打不开说明硬件太旧系统太旧。所以你必须自己换

**Reply by rong2er** [reply]
哦哦好的。我的电脑确实很旧了。。 那这个错误是不是也因为我电脑太旧的原因。我刚刚还去下了个新的FFMPEG  
![image](https://github.com/hooke007/MPV_lazy/assets/17141299/f93aa8d4-07e3-455e-a81d-3331266df497)
![image](https://github.com/hooke007/MPV_lazy/assets/17141299/3cb6b129-227b-489c-81e1-2b081f1eed28)

**Reply by hooke007** [reply]
hevc的硬解只有近年的硬件支持
我的项目也用不到ffmpeg

**Reply by rong2er** [reply]
好的。谢谢解答。。
